it/content/src/main/content/jcr_root/content/dam/formsanddocuments/core-components-it/samples/contentfragment/basic/_jcr_content/renditions/cq5dam.thumbnail.140.100.png.dir/.content.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you clean all the thumbnails and keep the collateral clean ?

Reply:
Removed